front/components/assistant_builder/shared.ts

```diff
@@ -30,6 +30,7 @@ export const CONNECTOR_PROVIDER_TO_RESOURCE_NAME: Record<
   github: { singular: "repository", plural: "repositories" },
   intercom: { singular: "element", plural: "elements" },
   webcrawler: { singular: "page", plural: "pages" },
+  zendesk: { singular: "ticket", plural: "tickets" },
```

Comment:
Is `ticket` the only available resource on Zendesk?

Reply:
No articles as well - I don't see a way of adding both wordings?

Reply:
We went with element/element for Intercom we can probably align?